format/spec.md

```diff
@@ -1132,15 +1132,15 @@ This serialization scheme is for storing single values as individual binary valu
 | **`long`**         | **`JSON long`**                           | `34`                                       | |
 | **`float`**        | **`JSON number`**                         | `1.0`                                      | |
 | **`double`**       | **`JSON number`**                         | `1.0`                                      | |
-| **`decimal(P,S)`** | **`JSON number`**                         | `14.20`                                    | Stores the decimal as a number with S places after the decimal |
+| **`decimal(P,S)`** | **`JSON string`**                         | `"14.20"`, `"2E+20"`                       | Stores the decimal string as-is |
```

Comment:
What does "as-is" mean? I think we need to be specific about the format here.

Reply:
@rdblue I added a detailed explanation of the string format, please take a look and let me know if there are any issues.